### ISSUE TITLE
Many fence lines occur after commit changes by git editor

### ISSUE THREAD

**Issue by nokia-git** [jesseduffield/lazygit#2294]
Many fence lines occur after commit changes by git editor

I'm using windows vscode with integrated terminal to access remote ssh server by linux.
lazygit is running on linux by vscode integrated terminal.

**To Reproduce**
1. type C to commit changes by git editor
2. my git eitor is nvim, after save and exit nvim, the lazygit is like the picture below

![unnamed](https://user-images.githubusercontent.com/80832836/203992108-4825ab42-a01c-4b2c-9388-70e833854489.png)

my lazygit is version 0.36.
version 0.35 is ok with the same actions.
and it is ok too when access ssh server by windows terminal.

**Comment by mark2185** [comment]
Yeah, it's a [bug](https://github.com/jesseduffield/lazygit/pull/2269) that was currently mitigated on current `master`, the temporary patch will remain in place until the underlying [vscode issue](https://github.com/xtermjs/xterm.js/issues/4238) is fixed (it may already have been fixed).

Until then, I advise you to either stay on `v0.35` or to build `lazygit` manually.

**Comment by floriansenat** [comment]
It's officially fixed since the release of VSCode 1.74.0 !
